### ISSUE TITLE
Staging Review [Facilities - Vet Centers] (WIP)

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#24175]
## Steps to complete Staging Review: 

- [ ] VFS Product Manager: Create this issue and fill in your team and feature name in the title.
- [ ] VFS: Link to this issue once created in [#vfs-platform-support](https://dsva.slack.com/channels/vfs-platform-support) in Slack; tag @vsp-product-support-members. 
- [ ] VSP: Schedule meeting with VSP reviewers and requesting team attendees (as listed below)
- [ ] VFS: At least 4 days before scheduled meeting, you must complete list of artifacts below
- [ ] VFS: Verify that access information has been provided for the tool or feature in staging. Do not put staging credentials in your va.gov-team ticket; store or reference them in a .md file in the va.gov-team-sensitive repository.
- [ ] VSP: Create feedback tickets and link to the Staging Review ticket (if no feedback, please comment stating that). 
- [ ] VFS: Respond to VSP feedback tickets.
- [ ] VFS: Update this ticket with the product’s launch date after it has gone live.
- [ ] VSP: Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=266151061).
- [ ] VFS: Close this ticket when the review is complete.

## Artifacts
- Link to [product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/facilities/vet-centers/product-brief.md)
- Link to the tool or feature in staging. Do not put staging credentials in your va.gov-team ticket; store or reference them in a .md file in the va.gov-team-sensitive repository.
-  Staging test-user information (~if~ _not applicable_)
-  Link to content brief (~if~ _not applicable_)
-  Link to or specify the artifact that represents this initiative's content source of truth (for example, mockup, prototype, Github, Word outline, or staging or preview link) -> CMS 
- Link to [release plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/product-management/release-plan-template.md) with the "Planning" sections completed (in each section: Phase I, Phase II, Go Live)
- Link to your product test cases/test plan in TestRail with test execution logs ([learn more](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/quality-assurance/qa-artifacts.md#test-plan)) _for QA practice area review_
- Link to your [Coverage for References](https://dsvavsp.testrail.io/index.php?/reports/view/12) and [Summary(Defects)](https://dsvavsp.testrail.io/index.php?/reports/view/14) reports in TestRail _for QA practice area review_
- Link to verification that accessibility checks are complete:
  - (preferred) Link to your [TestRail VSP accessibility test plan template](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc).
OR
  - Link to a doc in your product folder that indicates all required checks from the accessibility [staging review prep](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) doc are complete.
- List of known issues (with links to tickets when applicable)

## Meeting attendees from requesting team
- Product Manager (required): Michelle Middaugh
- DEPO Product lead (recommended): Dave Conlon
- DEPO Design lead (required): Ryan Thurlwell
- Anyone else from your team who significantly contributed to the artifacts provided (required): Leyda Hughes, Michael Pelz-Sherman, Gibran Rodriguez
- Content, IA, accessibility, and QA specialists (if applicable): Beth Potts, Josh Kim, Tze Lei

## Scheduling
What is your preferred time slot per the [VSP Collaboration Meetings Calendar](https://calendar.google.com/calendar/u/0/embed?src=adhocteam.us_4dn3o77gcm5e3vbiedlha96tc0@group.calendar.google.com&ctz=America/New_York)? 
**5/13 at 2 ET**
**5/14 at 2 ET**
 
Please note:
- All times are shown in ET. 
- Available time slots are indicated with [CALENDAR BLOCK]. 
- When you reserve a time slot, the reserved meeting will show up alongside the [CALENDAR BLOCK]. 
- Please select a time slot at least 4 business days from request date.

## Additional information
Please refer to the [VSP Collaboration Cycle README](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/README.md) on Github for more information about the Collaboration Cycle.

**Comment by shiragoodman** [comment]
hi @mmiddaugh  - Staging Review scheduled for 5/13 at 2:00pm ET

**Comment by mmiddaugh** [comment]
@shiragoodman we've had a conflict arise with our scheduled time but I spoke with @SKasimow about trading for her Friday slot.

**Comment by mmiddaugh** [comment]
Can we also please include Kevin Walsh from CMS?

**Comment by shiragoodman** [comment]
sure thing. I'll send an invite. also, per your request @mmiddaugh, meeting has been moved to Friday 5/14 at 2:00pm ET

**Comment by mmiddaugh** [comment]
@shiragoodman The content build separation process has impacted Tugboat (which will be the testing environment for Vet Centers) and changes still need to be merged to master. This will delay my testing process and the availability of artifacts. 

I apologize for the inconvenience. Can we please delay our staging review to Tuesday, May 21 at 11 or 11:30 ET?

**Comment by SKasimow** [comment]
@shiragoodman - if Michelle doesn't need the slot I'd like to have Friday back so Jen will be able to participate.

**Comment by shiragoodman** [comment]
hi @mmiddaugh  sorry I was OOO on Friday. Yes, we can move the review however Tuesday May 21 is not a date... can you please confirm when you'd like the review moved to?

@SKasimow yes, I'll move it back to Friday as soon as I hear from Michelle.

**Comment by AngelaFowler82** [comment]
Can I get an invite? Huge 
thanks.

**Comment by mmiddaugh** [comment]
@shiragoodman all artifacts are ready for review

Also - I was not able to test color contrast for these pages. The Colorblinding app is no longer available on Chrome. I'm happy to use another tool - suggestions?

**Comment by joshkimux** [comment]
@mmiddaugh No worries about color contrast checking, I'll handle that in the review!

**Comment by allison0034** [comment]
@mmiddaugh I currently cannot access these pages. Can you upload screen shots please?

**Comment by mmiddaugh** [comment]
[screencapture-preview-prod-vfs-va-gov-preview-2021-05-17-15_06_38.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/6496894/screencapture-preview-prod-vfs-va-gov-preview-2021-05-17-15_06_38.pdf)
[screencapture-preview-prod-vfs-va-gov-preview-2021-05-17-15_07_10.pdf](https://github.com/department-of-veterans-affairs/va.gov-team/files/6496896/screencapture-preview-prod-vfs-va-gov-preview-2021-05-17-15_07_10.pdf)


@allison0034 images are attached. Please note: these were taken from the preview server and images are not displayed as intended in that environment

**Comment by shiragoodman** [comment]
posting for awareness - scheduled for 5/20 at 2:00pm ET

**Comment by shiragoodman** [comment]
@peggygannon @allison0034 do you have feedback tickets to link here? I know you provided feedback at the meeting... just want to make sure Facilities has access to your ticket.

**Comment by peggygannon** [comment]
@shiragoodman Yes, here is my content ticket below and I just linked it to this epic. thx! 

https://app.zenhub.com/workspaces/platform---product-support-5f85b91c14d8df0018fac414/issues/department-of-veterans-affairs/va.gov-team/24857
